Kudu.Contracts/Functions/IKeyJsonOps.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Newtonsoft.Json.Linq;
+using System;
+
+namespace Kudu.Core.Functions
+{
+    public interface IKeyJsonOps<T>
+    {
+        int RequireKeyCount();
+        
+        // key generation is based on run time
+        string GenerateKeyUglyJson(Tuple<string,string>[] keyPairs, string functionRt, out string unencryptedKey);
```

Comment:
renamed, it was called ulglyjson as opposed to prettyjson (notepad++ json after formatting), uglyjson return a json string in with no whitespace or newlines